docs/dgx-pod.md

```diff
@@ -77,129 +79,130 @@ The DeepOps service container "DGXie" provides DHCP, DNS, and PXE services to th
    * Use the `ib_bond_addr` variable to configure the infiniband network adapters with IPoIB in a single bonded interface
 
-5. Configure cluster parameters by modifying the various yaml files in the `config/group_vars` directory. The cluster-wide global config resides in the `all.yml` file, while group-specific options reside in the other files. File names correspond to groups in the inventory file, i.e. `[dgx-servers]` in the inventory file corresponds with `config/group_vars/dgx-servers.yml`.
 
 ### 2. Prepare the management node(s)
 
 The configuration assumes a single cpu-only management server, but multiple management servers can be used for high-availability. Ensure that the inventory file is edited accordingly if using more than one management server.
 
-Install the latest version of Ubuntu Server 18.04 LTS on each management server. Be sure to enable SSH and record the user and password used during install.
+Install the latest version of Ubuntu Server 18.04 LTS on each management server. Be sure to enable SSH `apt install openssh-server` and record the user and password used during install.
+
 
-### 3. Deploy Kubernetes to the management node(s)
+### 3. Deploy Kubernetes to the management node(s) using Ansible and Kubespray
 
 Deploy Kubernetes to just the management node(s) using the k8s-cluster playbook:
 
 ```sh
+#NOTE:  If all the systems you are connecting to share the same username/password DeepOps will generate an ssh key and set up key auth on each node
+
+ansible-playbook -l kube-master playbooks/k8s-cluster.yml
+
+
 # NOTE: If SSH requires a password, add: `-k`
 # NOTE: If sudo on remote machine requires a password, add: `-K`
 # NOTE: If SSH user is different than current user, add: `-u ubuntu`
-ansible-playbook -l management,localhost -i config/inventory -b playbooks/k8s-cluster.yml
+
+ansible-playbook -l kube-master -k -u dgxuser -K playbooks/k8s-cluster.yml
+
 ```
 
-Test you can access the kubernetes cluster:
+Verify the Kubernetes cluster is running and you can access:
 
 ```sh
-# You may need to manually run: `sudo cp ./k8s-config/artifacts/kubectl /usr/local/bin`
-$ kubectl get nodes
-NAME      STATUS    ROLES         AGE       VERSION
-mgmt    Ready     master,node   7m        v1.12.4
-```
+# You may need to manually run: `sudo cp ./config/artifacts/kubectl /usr/local/bin`
 
-### 4. Deploy basic cluster services
+kubectl get nodes
 
-1. Install an ingress controller for the cluster.
+NAME             STATUS   ROLES    AGE   VERSION
+k8-mgmt01        Ready    master   23m   v1.12.5
+k8-mgmt02        Ready    master   21m   v1.12.5
+k8-mgmt03        Ready    master   21m   v1.12.5
 
-   See the [Ingress Guide](ingress.md) for details on how to install and configure ingress.
+```
 
-2. Deploy the internal apt package repository.
 
-   ```sh
-   kubectl apply -f services/apt.yml
-   ```
-   
-   Runs on port `30000`: http://mgmt:30000
+### 4. Deploy basic cluster services
```

Comment:
This section seems problematic. We are removing the registry, but not adding the k8s-services.yml playbook. Don't we want that? Also, don't we want to set up the ingress/loadbalancer beforehand? Right now it is in section 8.

Reply:
Edit: I see now what we've done. But it's a bit confusing - the k8s-services are not just for air-gapped environments. A customer might want the registry, container-replicator, etc, for other reasons. Recommend that we remove the "air-gapped" header below and include a note beforehand that the following k8s services are especially helpful for air-gapped environments.

Reply:
removed air-gapped heading and added note.  Also, moved the ingress controller backup, it was there before.  I must have moved too much during one edit.